js/pricing.js

```diff
@@ -9,17 +12,44 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirect = $('#redirect');
+        const $loader = $('.redirect-loader', '#redirect');
+        const $errorContainer = $('.redirect-error', '#redirect');
+        const $tryAgain = $('#try-again');
+
+        const orderUrl = "{{site.data.backend_links.orderUrl}}";
+        const prodApiUrl = "{{site.data.backend_links.prodApiUrl}}";
+        const devApiUrl = "{{site.data.backend_links.devApiUrl}}";
+        const registerTransactionPath = "{{site.data.backend_links.registerTransactionPath}}";
+        // `window.mode` we setup in the `getting-help/service-section.html` file by Jekyll.
+        const apiUrl = window.mode === "development" ? devApiUrl : prodApiUrl;
+
 
         $confirmPersonalInformation.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
         });
 
         $confirmDevelopmentAgreement.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
+        });
+
+        $orderButton.click(function() {
+            submitOrder();
+        });
```

Comment:
Can we simplify it?

```suggestion
        $orderButton.click(submitOrder);
```

Reply:
I added the [issue](https://github.com/SpineEventEngine/SpineEventEngine.github.io/issues/371) for reCAPTCHA logic implementation.